src/components/sign-attributes/sign-attribute-badge.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div class="sign-attribute">
+        <span>
+            <b-badge variant="secondary">{{attribute.attributeString}}: {{attribute.attributeValueString}}</b-badge>
+        </span>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { Artefact } from '@/models/artefact';
+import {
+    TextFragmentData,
+    TextFragment,
+    SignInterpretation,
+    ArtefactTextFragmentData,
+} from '@/models/text';
+import { EditionInfo } from '@/models/edition';
```

Comment:
Just looking quickly over this, why do we need to import Artefact, TextFragmentData, TextFragment, SignInterpretation, ArtefactTextFragmentData, and EditionInfo?  I only see InterpretationAttributeDTO used below.

Reply:
Ah, indeed. tslint can only detect unused imports along with unused variables. We turned this option off, since it doesn't recognize variable references inside the Vue templates, so if for example we have a component property that is only used inside the HTML, it complains that the property is unused.

I've enabled this in tslint and I'm going over all the imports now - I'll remove all unused ones and push. It will be a large commit, but just one commit for this, which you can probably ignore.